lib/widgets/picos_ink_well_button.dart

```diff
@@ -24,7 +24,8 @@ class PicosInkWellButton extends StatelessWidget {
   const PicosInkWellButton({
     required this.text,
     required this.onTap,
-    this.disabled = false,
+    required this.disabled,
+    this.enabledGrey = false,
```

Comment:
Please think of a way of extending on the default behavior, before you decide to alter it.
I would suggestion would look like this:

Instead of building in this switch, make an optional color parameter for changing each of the default colors.
This would expand the use cases this widget can be used for.